tests/e2e/test_token_details.py

```diff
@@ -1,6 +1,6 @@
 import unittest
 
-import duneapi.types
+import dune_client.types
```

Comment:
Nitpick suggestion:
```suggestion
from dune_client.types import Address
```

Reply:
Yes although I'll also have to change the line below. This was a sneaky one.